design/core_policies.md

```diff
@@ -0,0 +1,31 @@
+# Node.js core Security Policies
+
+The goal of that document is to design a behavior in Node.js core that could be used to disable certain features of
+Node.js.
+
+A simple way to reach that goal is to prevent the loading of certain core modules according to the defined policy.
+
+A policy is enforced by setting up a runtime flag for the Node.js process to disable a set of features:
+
+```sh
+$ node --disable-addons index.js
+```
+
+Also, a per core module blacklist can be set up:
+
+```
+$ node --disable=net,http,https,http2,dns,udp index.js
+```
+
+Question:
+* Do we want to allow policy definition from a file
```

Comment:
are we talking about the:

* requested policies for a given package being put into a file
* the flags enabling features to be put into a file
* the authorized pre-authorized permissions for features being placed into a file

It seems like we might need more than 1 file and/or need to clarify which parts need to live where